Magento_Checkout/templates/onepage/link.phtml

```diff
@@ -5,7 +5,7 @@
         title="<?= $block->escapeHtmlAttr(__('Proceed to Checkout')) ?>"
         data-mage-init='{
             "Magento_Checkout/js/proceed-to-checkout": {
-                "checkoutUrl":"<?= $block->getCheckoutUrl(); ?>"
+                "checkoutUrl":"<?= $block->escapeUrl($block->getCheckoutUrl()); ?>"
```

Comment:
`"checkoutUrl":"<?= $block->escapeJs($block->escapeUrl($block->getCheckoutUrl())) ?>"`
as in orignal file